src/bmconfigparser.py

```diff
@@ -73,8 +83,9 @@ def get(self, section, option, raw=False, variables=None):
             return ConfigParser.ConfigParser.get(
                 self, section, option, True, variables)
         except ConfigParser.InterpolationError:
-            return ConfigParser.ConfigParser.get(
-                self, section, option, True, variables)
+            return ConfigParser.ConfigParser.get(self, section, option, True, variables)
+        except TypeError:
+            return ConfigParser.ConfigParser.get(section, option, variables)
```

Comment:
This could also throw an exception, if the default is missing, so it's incomplete. Why not use `super` instead, wouldn't that make it easier to write code that runs on both python2 and python3?